### DISCUSSION TITLE
Install problem

### DISCUSSION THREAD

**Discussion by LionelPerret** [qax-os/excelize#2005] [Q&A]
Hi

I'm working in the VS Code Terminal

go get github.com/xuri/excelize
   doesn'r work ('go get' is no longer supported outside a module.) and I need to do :
go install github.com/xuri/excelize@latest
   instead

And then I have the issue :
     version constraints conflict:
        github.com/360EntSecGroup-Skylar/excelize/v2@v2.8.1: parsing go.mod:
        module declares its path as: github.com/xuri/excelize/v2
                but was required as: github.com/360EntSecGroup-Skylar/excelize/v2
                
Thanks for your help!

**Comment by xuri** [comment]
Please don't create duplicate issue #2006. I have already replied to that issue. Please read the README or documentation. Install lasted version by the following command:

```bash
go get github.com/xuri/excelize/v2
```

I closed this, if you have any questions, please let me know, and you can reopen this anytime.
